sos/policies/cos.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import with_statement
+
+from sos.plugins import CosPlugin
+from sos.policies import PackageManager, LinuxPolicy
+
+import os
+
+
+class CosPolicy(LinuxPolicy):
+    distro = "Container-Optimized OS"
+    vendor = "Google Cloud Platform"
+    vendor_url = "https://cloud.google.com/container-optimized-os/"
+    valid_subclasses = [CosPlugin]
+    PATH = "/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin"
+
+    @classmethod
+    def check(cls):
+        try:
+            with open('/etc/os-release', 'r') as fp:
+                lines = fp.readlines()
+                for line in lines:
+                    if line.startswith("ID="):
+                        return line.split("=")[1].strip() == "cos"
+            return False
+        except IOError:
+            return False
+
```

Comment:
You seem to have some missing bits here: you're importing `PackageManager`, but then not using it anywhere. This means that package name triggers and package verification for plugins won't work with this policy. Normally this is initialised inside the policy `__init__()` method so that policy -> package manager callbacks from plugin code work correctly.

Setting a `name_pattern` will also allow you to control default report naming templates.

Reply:
Good catch. Yeah I should not import `PackageManager`, because COS actually do not have a package manager, similar to what CoreOS does.